tcl.html.markdown

```diff
@@ -50,6 +52,10 @@ syntax of other languages, allows it to fade into the background of whatever
 domain-specific functionality a project needs. It's syntax, which is even
 lighter that that of Lisp, just gets out of the way.
 
+There is a document floating around the web titled "Why you should not use
+Tcl".  Ignore that.  It's woefully out-of-date, and even when it was published
+was mostly FUD.
+
```

Comment:
Most of the formatting/typo changes seem OK, but isn't this statement a bit unnecessary for an introduction?